codeformatter/eclipse/icellmobilsoft-eclipse-code-formatter.xml

```diff
@@ -4,7 +4,7 @@
         <setting id="org.eclipse.jdt.core.formatter.insert_space_after_ellipsis" value="insert"/>
         <setting id="org.eclipse.jdt.core.formatter.insert_space_after_comma_in_enum_declarations" value="insert"/>
         <setting id="org.eclipse.jdt.core.formatter.insert_space_before_comma_in_allocation_expression" value="do not insert"/>
-        <setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_for_statment" value="common_lines"/>
+        <setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_for_statment" value="preserve_positions"/>
```

Comment:
release note is missing

Reply:
Ha jol latom ez a megoldas a "for" metodusok tordeleset erinti, nem az issueban leirt problemat kezelne. Nem inkabb a `<setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_enum_constant_declaration" value="separate_lines_if_not_empty"/>` kellene?

Reply:
Ez a pull nem csak az enumoknál való formázásról szól, hanem úgy általánosságban, ahol felsorolva vannak paraméterek, ott kierőszakolja a 160 karakteres sor végéig tartó felsorolást, ami sokszor átláthatóbbá tehető, ha logikai csoportok mentén új sorokba kezdjük az összetartozó paramétereket. Az enumos példa csak egy szemléltetés, hogy például ott is csúnyává teszi a kódot.

A parentheses_positions_in_enum_constant_declaration paraméter nem segít ebben, ha berakom amit írtál ilyen lesz:
```java
    ALPHA(
            EnumSet.of(ProductType.ALPHA_LONG_NAMED_ENUM_ONE, ProductType.ALPHA_LONG_NAMED_ENUM_TWO)
    ), BETA(
            EnumSet.of(
                    ProductType.BETA_LONG_NAMED_ENUM_ONE,
                    ProductType.BETA_LONG_NAMED_ENUM_TWO,
                    ProductType.BETA_LONG_NAMED_ENUM_THREE,
                    ProductType.BETA_LONG_NAMED_ENUM_FOUR,
                    ProductType.BETA_LONG_NAMED_ENUM_FIVE)
    ), GAMMA(
            EnumSet.of(ProductType.GAMMA_LONG_NAMED_ENUM_ONE, ProductType.GAMMA_LONG_NAMED_ENUM_TWO, ProductType.GAMMA_LONG_NAMED_ENUM_THREE)
    );
```

Reply:
Igen az issue leirasa kevert be hogy mi a cel, igy oke. Viszont a formatter valtozasrol kellene leiras mi valtozott.

Reply:
pótoltam a release notes-ot